index.html

```diff
@@ -405,8 +405,278 @@ <h4 class="title-small">Winning the talent war with a scalable tech skills platf
             <button class="skilling-button">HOW THEY'RE SKILLING UP</button>
         </div>
     </section>
+    <!--Follow us section-->
+    <section class="follow-us-section">
+        <div class="Follow-us-section-header">
+            <h6 class="title-pink">FOLLOW US</h6>
+            <h3 class="title-sub">Share your skills</h3>
+        </div>
+        <div class="follow-us-social-share-container">
+            <div>
+                <img src="/aby-website/images/chevron-left-social.svg" alt="">
+            </div>
+            <div class="social-share-window">
```

Comment:
These are cards, so you should call them "social-share-cards". It would make more sense contextually than "window"